src/main/java/seedu/allonus/modules/StudyManager.java

```diff
@@ -106,6 +110,77 @@ public void deleteModule(String userInput) {
 
     }
 
+    public void editModule(String userInput, TextUi ui) {
+        try {
+            String moduleIndexString = userInput.replace("edit ", "");
+            int moduleIndex = Integer.parseInt(moduleIndexString) - 1;
+            if (modulesList.get(moduleIndex) != null) {
+                Module moduleToEdit = modulesList.get(moduleIndex);
+                printMessage("Here is the module that you have chosen to edit:");
+                printMessage(moduleToEdit.toString());
+                printMessage("Choose the part that you would like to edit: ");
```

Comment:
You could consider refactoring out the string constants here! And likewise for the subsequent strings too